### PR TITLE
Ultra

### PR DESCRIPTION
**Pull request by adreena**
I'm adding ultra changes under smarts/ultra

### CODE REVIEW

File: smarts/ultra/worker.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import logging
```
Comment (Gamenot) [inline smarts/ultra/worker.py:1]: What is the purpose of this file?

Comment (adreena) [reply smarts/ultra/worker.py]: @Gamenot removed them

**Review by AlexLewandowski — COMMENTED** [review]
there are still references to smarts.ultra.registry in train.py and evaluate.py

**Comment by Gamenot** [comment]
@AlexLewandowski , thank you for spotting that.

**Pull request merged**